src/main/java/com/spring/board/springboard/domain/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,34 @@
+package com.spring.board.springboard.domain;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+import java.util.List;
+
+@RestControllerAdvice
+public class GlobalExceptionHandler {
+
+
+    @ExceptionHandler(MethodArgumentNotValidException.class)
+    public ResponseEntity<List<ValidationErrorResponse>> handleDtoValidationException(MethodArgumentNotValidException methodArgumentNotValidException){
+        List<ValidationErrorResponse> responses = methodArgumentNotValidException.getBindingResult()
+                .getFieldErrors()
+                .stream()
+                .map(fieldError -> new ValidationErrorResponse(
+                        fieldError.getField(),
+                        fieldError.getDefaultMessage()
+                )
+                ).toList();
+
+        return ResponseEntity.badRequest().body(responses);
```

Comment:
```suggestion
        return ResponseEntity.badRequest()
          .body(responses);
```
객체지향 생활체조, 한줄에 한점만 찍는다 적용해주세요~